action.yml

```diff
@@ -1,4 +1,4 @@
-name: 'Merge Conflict Finder'
+name: 'Merge Conflict Finder 2'
```

Comment:
Could you remove the number 2 added here?

Reply:
Fixed